templates/flashes_helper.rb

```diff
@@ -0,0 +1,5 @@
+module FlashesHelper
+  def user_facing_flashes
+    flash.to_hash.slice("alert", "error", "notice", "success")
```

Comment:
@derekprior do you mean grab the list of user facing flash keys from Rails? I couldn't find in https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/middleware/flash.rb a way to do that.